### ISSUE TITLE
geom_crossbar: bad label for y axis

### ISSUE THREAD

**Issue by ASmirnov-HORIS** [JetBrains/lets-plot#326]
Look at the label near the y axis:
```python
from lets_plot import *
LetsPlot.setup_html()
data = {
    'x': ['a', 'b', 'c'],
    'ymin': [5, 6, 4],
    'ymax': [7, 9, 8],
}
ggplot(data) + geom_crossbar(aes(x='x', ymin='ymin', ymax='ymax'))
```
<img src="https://user-images.githubusercontent.com/26162403/108082061-b8989980-7082-11eb-8cb5-87b732d427dd.png" width="400" height="300"/>

**Comment by ASmirnov-HORIS** [comment]
The same is true for `geom_errorbar()`.

**Comment by alshan** [comment]
Fixed in v2.0.1.
